docs/reference/nats.md

```diff
@@ -0,0 +1,68 @@
+---
+title: 'NATS Subjects'
+date: 2024-10-04
+sidebar_position: 12 
+draft: false 
+---
+
+This reference article attempts to give a quick overview of NATS subjects used by wasmCoud.
+
+## wRPC
+
+NATS Subjects used for wRPC communication uses the following subject pattern:
+
+```
+{lattice}.{component_id}.wrpc.{protocol_version}.{namespace}:{package}/{interface}@{version}.{function}
+```
+
+[See RPC article for more details](/docs/hosts/lattice-protocols/rpc)
+
+## Control Interface
+
+The complete Control Interface documentation can be [found here](/docs/hosts/lattice-protocols/control-interface), and follows the following pattern: e
+
+```
```

Comment:
```suggestion
```plaintext
```

Reply:
Resolved in b4551570d12304f59811dbd7eec229a3e9ef5f15